docs/TRANSLATIONS.MD

```diff
@@ -0,0 +1,58 @@
+# 🐍 🌍 🚀Translation Release Strategy
+Thanks for your interest in translating the micro:bit Python Editor. Translations for all the Micro:bit Educational Foundation’s editors and apps are contributed by an amazing team of volunteers and we’d love for you to be a part of this too. 
+
+This document sets out the strategy and process for how the Foundation manages the translation of the Python Editor hosted at python.microbit.org
+
+## 1. Translation and Management in the Python Editor
+
+Translation is the act of taking the written materials in one language and converting them into another language in the most meaningful way possible. In terms of the micro:bit Python Editor, translation happens on the [strings marked for translation in the projects' codebase](https://github.com/bbcmicrobit/PythonEditor/lang/en.js). The default language of the editor is English and you may select another available language using the Globe icon in the menu.
+
+**1.1 Crowdin**
```

Comment:
These are rendered inline:

![image](https://user-images.githubusercontent.com/29712657/64617168-e5e51080-d3d5-11e9-8a94-2682130fa200.png)

Any reason not to use `###`?

Reply:
I think Paper Markdown (which i exported this from) may be different from Github markdown